src/main/java/ev3dev/actuators/lego/motors/BaseRegulatedMotor.java

```diff
@@ -165,18 +165,18 @@ public void backward(){
      * and the position of the motors will not be maintained.
      */
     @Override
-    public void flt(boolean b) {
-        this.flt();
+    public void flt(boolean immediateReturn) {
+        doStop(COAST, immediateReturn);

```

Comment:
This is consistent with leJOS:
flt() sends a move command to the regulator class: https://sourceforge.net/p/lejos/ev3/code/ci/master/tree/ev3classes/src/lejos/hardware/motor/BaseRegulatedMotor.java#l171
Kernel regulator interface handles it here: https://sourceforge.net/p/lejos/ev3/code/ci/master/tree/ev3classes/src/lejos/internal/ev3/EV3MotorPort.java#L427
```
float speed = 0,
int acceleration = <current acceleration>,
int limit = no limit,
boolean hold = FALSE,
boolean waitComplete = <forwarded>
```